src/app/server/Mdns.cpp

```diff
@@ -250,11 +251,14 @@ void StartServer()
     ChipLogProgress(Discovery, "Start dns-sd server");
     CHIP_ERROR err = chip::Mdns::ServiceAdvertiser::Instance().Start(&chip::DeviceLayer::InetLayer, chip::Mdns::kMdnsPort);
 
-    // TODO: advertise this only when really operational once we support both
-    // operational and commisioning advertising is supported.
-    if (GetCurrentNodeId() != kUndefinedNodeId)
+    err = app::Mdns::AdvertiseOperational();
+    if (err != CHIP_NO_ERROR)
+    {
+        ChipLogError(Discovery, "Failed to advertise operational identities: %s", chip::ErrorStr(err));
```

Comment:
```suggestion
        ChipLogError(Discovery, "Failed to advertise operational node: %s", chip::ErrorStr(err));
```
For consistency with the previous log message.